data/transport missions.txt

```diff
@@ -3404,3 +3404,69 @@ mission "Capture Smuggler"
 		payment 150000
 		conversation
 			`The huge man drags the unconscious smuggler out of the cargo container with one hand and carries the cargo box with the other. He stops in front of you. "<first> <last>," he says. "My employer, he will be very happy. Your payment, you will receive soon. Is it not good, to be reliable?" The man leaves your ship, letting the smuggler's legs scrape along the ground as he walks. A moment later, you receive <payment> from an anonymous account.`
+
+
+
+mission "Inheritance Redirection"
+	description "Travel to <destination> to act as heir for the late warlord Limping Pappa."
+	source
+		government "Pirate"
+	destination
+		distance 7 9
+		government "Republic" "Free Worlds" "Syndicate"
+	to offer
+		credits >= 400000
+	on accept
+		payment -200000
+	on offer
+		conversation
+			`You notice a computer terminal blinking in the corner of an alley.`
+			choice
+				`	(Check it out.)`
+				`	(Ignore it.)`
+					decline
+			`When you approach, you are greeted by a voice from the terminal; "Dear Sir, Madam, please respond!`
+			choice
+				`	"Hello"`
+				`	(Ignore it and walk away.)`
+					decline
+			`	"My name is Mr. Smith and I'm contacting you with a highly confidential and urgent proposal. I am, or more accurately was, the financial manager of the recently deceased warlord Limping Pappa. I am looking for a highly trustworthy individual who can help me relocate Mr. Pappa's fortune."`
+			`	"Mr. Pappa acquired over 300 million credits."`
+			choice
+				`	"How did Mr. Pappa acquire his fortune exactly?"`
+				`	"What are you proposing?"`
+					goto proposal
+				`	"This sounds like a credits-stealing scam, I will not fall for this"`
+					decline
+			`	"Oh, euuh, Mr. Pappa was specialized into tricking rich victims to transfer credits to him.`
+			label proposal
+			`	"I obviously cannot make myself a heir to Mr. Pappa. So what I propose is to present you as heir and have the inheritance transferred to you.`
+			`	"When you receive the inheritance on <destination>, then you send half of it back to me."`
+			choice
+				`	"Okay, let's do this."`
+				`	"Sounds too good to be true, what is the catch?"`
+				`	"Sorry, this sounds too illegal for me."`
+					decline
+			`	Mr. Smith responds: "There is one small problem with the inheritance tax, it needs to be paid upfront. Can you pay 200,000 credits so that I can get the process going?"`
+			choice
+				`	"Sure, no problem."`
+					goto pay_upfront
+				`	"Are you trying to trick me?"`
+				`	"Sounds too risky, I don't want to pay anything up-front."`
+					goto no_deal
```

Comment:
I'd say try to use one-word labels since that's the norm for the rest of the game, so `pay` instead of `pay_upfront` and `no` instead of `no_deal` for example.

Reply:
Done